Cargo.toml

```diff
@@ -40,9 +40,9 @@ net = []
 
 [dependencies]
 log = "0.4.8"
+libc = "0.2.86"
```

Comment:
Revert this, not needed for this change. If needed you can make the example unix only.

Reply:
Can I add libc in [target.'cfg(windows)'.dev-dependencies]?
If not acceptable, I would like to know how to make the example unix only.

Reply:
Thanks! I will try it.